builder/frameworks/arduino/arduino-samd.py

```diff
@@ -170,6 +177,33 @@
             os.path.join(FRAMEWORK_DIR, "cores", BUILD_CORE, "api", "deprecated-avr-comp")
         ]
     )
+elif VENDOR_CORE == "clearcore":
+    env.Prepend(
+        CPPPATH=[
+            os.path.join(FRAMEWORK_DIR, "variants", "clearcore", "ThirdParty", "SAME53", "CMSIS", "Device", "Include"),
+        ]
+    )
+
+    env.Append(
+        CPPPATH=[
+            os.path.join(FRAMEWORK_DIR, "cores", "arduino", "api"),
+            os.path.join(FRAMEWORK_DIR, "cores", "arduino"),
+            os.path.join(FRAMEWORK_DIR, "variants", "clearcore"),
```

Comment:
You don't need this line because the code below it that checks for "is there a build.variant and if yes add it to the cpp path" already takes care of this: https://github.com/platformio/platform-atmelsam/blob/17187308c7b98e748e06e7b1fed615d3af54662c/builder/frameworks/arduino/arduino-samd.py#L219-L221